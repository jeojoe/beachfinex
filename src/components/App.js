import React from 'react';
import { connect } from 'react-redux';

import Ticker from './Ticker';
import Book from './Book';
import Trades from './Trades';

const App = () => (
  <div className="App">
    <header className="App-header">
      <Trades />
      <Ticker />
      <Book />
      <a
        className="App-link"
        href="https://reactjs.org"
        target="_blank"
        rel="noopener noreferrer"
      >
        Learn React
      </a>
    </header>
  </div>
);

function mapStateToProps(state) {
  return {
    ticker: state.ticker,
  };
}

export default connect(mapStateToProps)(App);
