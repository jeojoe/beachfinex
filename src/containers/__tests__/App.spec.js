import React from 'react';
import { shallow } from 'enzyme';

import App from '../App';

describe('<App />', () => {
  it('renders correctly', () => {
    const wrapped = shallow(<App />);
    expect(wrapped).toMatchSnapshot();
  });
});
