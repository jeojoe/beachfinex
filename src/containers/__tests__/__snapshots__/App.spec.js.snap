// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<styled.div>
  <styled.h1>
    Beachfinex
  </styled.h1>
  <styled.div>
    <Connect(WithWebSocket) />
  </styled.div>
  <div
    className="row"
  >
    <div
      className="col-2"
    >
      <styled.div>
        <Connect(WithWebSocket) />
      </styled.div>
    </div>
    <div
      className="col-1"
    >
      <styled.div>
        <Connect(WithWebSocket) />
      </styled.div>
    </div>
  </div>
</styled.div>
`;
